playground/backend/ExampleData.scala

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
I think we should move this file from the top level of the backend folder, if we only need this in the `life_cycle`, let's move it there, what do you think?

Reply:
Done